.github/workflows/full_kubeflow_integration_test.yaml

```diff
@@ -79,8 +79,31 @@ jobs:
         while ! curl localhost:8080; do echo waiting for port-forwarding; sleep 1; done; echo port-forwarding ready
     - name: test dex login
       run: |
+        # Install requirements
         pip3 install requests
-        ./tests/gh-actions/test_dex_login.py
```

Comment:
Please change it back to the original script. It should be minimal, reusable and just fail the whole pipeline immideately.

Reply:
If you want to add something please add it in the script file test_dex_login.py